sky/spot/controller.py

```diff
@@ -234,7 +237,8 @@ def run(self):
                 spot_state.set_failed(
                     self._job_id,
                     failure_type=spot_state.SpotStatus.FAILED_CONTROLLER,
-                    failure_reason='Unexpected error occurred.')
+                    failure_reason='Unexpected error occurred. For details, '
+                    f'run: sky spot logs --controller {self._job_id}')
```

Comment:
nit: can add parens around the string to auto format

Reply:
Done. Thanks!